carma-messenger-ui/website/index.html

```diff
@@ -1,151 +1,136 @@
+    <script type="text/javascript" src="thirdparty/jqwidgets/jqxbuttons.js"></script>
+    <script src="thirdparty/bootstrap-4.4.1-dist/js/bootstrap.min.js"></script> 
+    <script src="thirdparty/bootstrap-switch-button-1.1.0/dist/bootstrap-switch-button.min.js"></script>
+    <script src="thirdparty/ros/eventemitter2.min.js"></script>
+    <script src="thirdparty/ros/roslib.min.js"></script>
 
-    <!-- <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
-  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script> -->
+    <!--START: custom css-->
+    <link rel="stylesheet" type="text/css" href="styles/index.css" />
+    <link rel="stylesheet" href="thirdparty/jqwidgets/styles/jqx.base.css" type="text/css" />
+    <link rel="stylesheet" href="thirdparty/jquery-ui-1.12.1/jquery-ui.css" type="text/css" />
+    <!--END: custom css-->
 
+    <!--START: CUSTOM scripts-->
+    <script src="scripts/carma.config.js"></script>
+    <script src="scripts/index.js"></script>
+     <!--END: CUSTOM scripts-->
 
-    <!--END: Thirdparty-->
 </head>
 
 <body>
-    <div id="messenger-main-div">
+    <noscript>
+        <h2 style="color: #ff0000">
+            Seems your browser doesn't support Javascript! Websocket relies on Javascript being enabled. Please enable Javascript and reload this page!
+        </h2>
+    </noscript>
+
+   <div id="messenger-index-div">
+        <!--Messenger Title Div--> 
         <div class="MessengerTitleFrame">
-            <div class="MessengerTitleRec">
-                <div class="Messenger_back_Div">
-                    <img class="Messenger_back_arrow" id="Messenger_back_arrow" src="images/Messenger_Back_Arrow.png"
-                        onclick="javascript:history.go(0);this.style.display='none'">
-                </div>
-                <img class="logo" src="images/C_Messenger@2x.png" alt="logo1 Image Not Available">
-                <!-- <img class="logo2" src="images/CARMA_color.png" alt="Logo2 Image Not Available">
-                <img class="Messenger" src="images/messenger_Title.png" alt="Messenger Image Not Available"> -->
-            </div>
-
-        </div>
-        <div id="divSystemLogs" hidden>
-            <div id="statusIndicator">
-                <p id="connecting">
-                    Connecting to ROSbridge...<br>
-                </p>
-                <p id="connected" style="color:#00D600; display:none">
-                    Connected to ROSBridge.<br>
-                </p>
-                <p id="error" style="color:#FF0000; display:none">
-                    ROSBridge unable to connect.<br>
-                </p>
-                <p id="closed" style="display:none">
-                    ROSBridge connection closed.<br>
-                </p>
+            <div class="MessengerTitleRec">     
+            <div class="messenger-logo-div">
+                <img class="logo" src="images/C_Messenger@2x.png" />
```

Comment:
Could you please go ahead and delete the logos we probably will not use anymore. The rest you can keep if there's a chance we will use it in the future implementations. 
> messenger_icon.png
> messenger_title.png
> carma_logo.png

Reply:
Addressed